modules/preparing-for-installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+[id="preparing-install-microshift-from-rpm-package_{context}"]
+= Preparing to install MicroShift from an RPM package
+
+Before installing MicroShift from an RPM package, you must configure your {op-system} machine to have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.
+
+MicroShift uses the Open Data Foundation Logical Volume Manager (ODF-LVM) Container Storage Interface (CSI) provider for storing PVs. ODF-LVM relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG in which ODF-LVM can create the LVM Logical Volumes (LVs) for your workload's PVs.  
+
+The simplest way to configure an LVM VG that allows ODF-LVM to create the LVM LVs for your workload's PVs is to set your machine to reserve capacity during the installation of {op-system} on your machine. 
+
+.Prerequisites 
+
+* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met. 
+
+.Procedure 
+
+. In the graphical installer under *Storage Configuration*, select *Custom* -> *Done* to open the dialog for configuring partitions and volumes. 
+
+. Under *New Red Hat Enterprise Linux 8.x Installation*, select *Click here to create them automatically*. 
+
+. Select the root partition, */*, and reduce *Desired Capacity* so that the VG has sufficient capacity for your PVs, then click *Update Settings*. 
+
+. Complete your installation. 
++
+[NOTE]
+====
+For more options on partition configuration, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/performing_a_standard_rhel_8_installation/index#manual-partitioning_graphical-installation[Configuring Manual Partitioning]. 
+====
+
+. After installation, verify the VG capacity available on your system by entering the following command: 
++
+[source,terminal]
+----
+$ sudo vgs
+----
++
+Example output:
++
+[source,terminal]
+----
+VG   #PV #LV #SN Attr   VSize    VFree
+rhel   1   2   0 wz--n- <127.00g 54.94g
+----
++
+[NOTE]
+====
+* The name of the default VG, for example, `rhel`, must be stored, as it is used during the installation of MicroShift. 
```

Comment:
Maybe to clarify: If the default VG name isn't 'rhel', then the user must make changes as part of the "installing MicroShift from RPM" procedure later using the user-configured VG name.
"must be stored" (to me) sounds like storing it in a file somewhere.

Reply:
Maybe we want to say something more like "Remember this name..."?

We could also simplify the instructions by removing the conditional statement and _always_ suggest that they create the lvmd.conf file. At that point in the instructions we would tell them to provide the name of the volume group where MicroShift workloads can create PVs.

Reply:
The name of the volume group will not be "rhel" unless they make it so explicitly when installing the OS. Let's give them a minimal lvmd.conf file.

@copejon can you help with that?

Reply:
Users need to select “create partitions automatically” to have an LVM system, but then “rhel” is the default.

Reply:
Ah, I did not realize that. I thought it used the hostname to set a unique VG name.

Either way, let's streamline this and remove conditionals. We've told them to click to create the filesystems automatically. That should mean we can remove this note, here. We can add a separate procedure for more complex setup with custom names later.

Reply:
Note removed.